src/main/java/io/pinecone/PineconeIndexOperationClient.java

```diff
@@ -0,0 +1,31 @@
+package io.pinecone;
+
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClient;
+
+import java.io.IOException;
+
+public class PineconeIndexOperationClient {
+
+    private AsyncHttpClient client;
+
+    PineconeIndexOperationClient(AsyncHttpClient client) {
+        this.client = client;
+    }
+
+    public void deleteIndex(String environment, String apiKey, String indexName) throws IOException {
```

Comment:
Could we move the `environment` and `apiKey` params to the client constructor so this method only takes the `indexName`?

Reply:
Yes, my plan is to use two existing config objects and remove any parameters from deleteIndex method. Let me actually make that change now.